### PR TITLE
LPS-136040 - Tags do not display in Tags selector for Account

### PR DESCRIPTION
**Pull request by wanderlast**
From @Odrakir1:

> This issue was caused by a missing group id, in this case, the Global group id. The asset was being fetched by the Control Panel's group id only, adding the Global group id in the search solved the problem.
> 
> 
> 
> https://issues.liferay.com/browse/LPS-136040

### CODE REVIEW

**Comment by wanderlast** [comment]
ci:test:sf

File: modules/apps/asset/asset-tags-selector-web/src/main/java/com/liferay/asset/tags/selector/web/internal/display/context/AssetTagsSelectorDisplayContext.java [hunk 1]
```diff
@@ -170,13 +170,17 @@ public PortletURL getPortletURL() {
...
+			(ThemeDisplay)_httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
 
+		if (ArrayUtil.isEmpty(_groupIds)) {
 			_groupIds = new long[] {themeDisplay.getScopeGroupId()};
 		}
+		else {
+			_groupIds = ArrayUtil.append(
+				_groupIds, themeDisplay.getScopeGroupId());
+		}
```
Comment (drewbrokke) [inline modules/apps/asset/asset-tags-selector-web/src/main/java/com/liferay/asset/tags/selector/web/internal/display/context/AssetTagsSelectorDisplayContext.java:183]: `StringUtil.split` will always return an array, so you can simplify this by removing the conditional and always using `ArrayUtil.append`.

**Review by drewbrokke — COMMENTED** [review]
@Odrakir1 @wanderlast this seems ok to me, just one comment. Also, our team does not own the component you're changing, so I'd request approval from @ealonso's team as well.

**Comment by ealonso** [comment]
Hey @wanderlast @Odrakir1,

Could you please send this one to @liferay-echo user?

Regards,
Eudaldo

**Comment by wanderlast** [comment]
Thanks for taking a look at this. I'll forward to liferay-echo.

**Pull request closed without merging**